src/main/java/org/ays/auth/exception/AysUserNotPassiveException.java

```diff
@@ -1,15 +1,13 @@
 package org.ays.auth.exception;
 
-import org.ays.auth.model.enums.AysUserStatus;
-import org.ays.common.exception.AysNotExistException;
-
 import java.io.Serial;
+import org.ays.common.exception.AysConflictException;
 
 /**
  Exception thrown when a user does not in a passive state.
- * This exception extends {@link AysNotExistException}.
+ * This exception extends {@link AysConflictException}.
  */
-public final class AysUserNotPassiveException extends AysNotExistException {
+public final class AysUserNotPassiveException extends AysConflictException {
```

Comment:
Aşağıdaki değişikliği yaptıktan sonra `serialVersionUID` değerinin yenilenmesi gerekiyor
```suggestion
public final class AysUserAlreadyActiveException extends AysConflictException {
```